gemmbench/gemm_bench.py

```diff
@@ -36,9 +36,9 @@ def compile_gemm(tag, config, kernel_dir, vmfb_dir, target, extra_compiler_args,
     parser.add_argument("--target", help="The IREE hip target to compile for", type=str, default="gfx942")
     parser.add_argument(
         "--Xiree_compile",
-        action='append',
+        nargs='+',
         default=[],
-        help="Extra command line arguments passed to the IREE compiler. This can be specified multiple times to pass multiple arguments."
+        help="Extra command line arguments passed to the IREE compiler. The flags need to be specified without the `--` or `-`"
     )
     parser.add_argument(
```

Comment:
If you change this one one, I think we should also change the two flags below to use `nargs=+`

Reply:
maybe... i dont know where all it is used. Could be a follow up.